engines/glk/zcode/infocom6x8.xbm

```diff
@@ -0,0 +1,42 @@
+#define infocom6x8_width 192
```

Comment:
Could you append a `.h` extension to xbm files?
It's better as they are not xbm now.

Reply:
This is a valid XBM file - they're designed to allow including them into C/C++ source code, which is why I chose it instead of BMP.

Reply:
Woah! I didn't know! Thanks for the explanation